### PR TITLE
test_: unique project per run

### LINKED ISSUE #5980: Functional tests running on `develop` branch don't report failure
Here's a PR job, properly failing: https://ci.status.im/blue/organizations/jenkins/status-go%2Fprs%2Ftests-rpc/detail/PR-5969/15/pipeline/

Image

And here is the job for `develop` branch getting same error, but passing:

https://ci.status.im/blue/organizations/jenkins/status-go%2Ftests-rpc-develop/detail/tests-rpc-develop/7/pipeline/

Image

-----

So it looks like all jobs are passing. But in fact all of them have failed 🙁 

Image

### PR DESCRIPTION
**Pull request by antdanchenko**
fixes 2 issues:

1. `Error response from daemon: Conflict. The container name "/964f2dd64c34_tests-functional-status-go-1" is already in use by container "0fe8766cc6c9e155217924d13ef196e97094eef2b67978d15901a928acde5309". You have to remove (or rename) that container to be able to reuse that name.`

2. second session attaches to the first instead of rebuilding containers with status-backend when a session is already running and another session starts

3. Fixes https://github.com/status-im/status-go/issues/5980

### CODE REVIEW

File: _assets/scripts/run_functional_tests.sh [hunk 1]
```diff
@@ -24,29 +24,36 @@ mkdir -p "${merged_coverage_reports_path}"
 mkdir -p "${test_results_path}"
 
 all_compose_files="-f ${root_path}/docker-compose.anvil.yml -f ${root_path}/docker-compose.test.status-go.yml"
+timestamp=$(date +%s)
 
 # Run functional tests
 echo -e "${GRN}Running tests${RST}, HEAD: $(git rev-parse HEAD)"
-docker-compose ${all_compose_files} up -d --build --remove-orphans
+docker-compose -p ${timestamp} ${all_compose_files} up -d --build --remove-orphans
```
Comment (jakubgs) [inline _assets/scripts/run_functional_tests.sh]: How about being a bit more concrete?
```suggestion
docker-compose -p "status-go-func-tests-${timestamp}" ${all_compose_files} up -d --build --remove-orphans
```

File: _assets/scripts/run_functional_tests.sh [hunk 2]
```diff
@@ -24,29 +24,36 @@ mkdir -p "${merged_coverage_reports_path}"
...
 
 echo -e "${GRN}Running tests-rpc${RST}" # Follow the logs, wait for them to finish
-docker-compose ${all_compose_files} logs -f tests-rpc > "${root_path}/tests-rpc.log"
+docker-compose -p ${timestamp} ${all_compose_files} logs -f tests-rpc > "${root_path}/tests-rpc.log"
```
Comment (jakubgs) [inline _assets/scripts/run_functional_tests.sh]: You should be using Docker Compose V2 and not V1, by using `docker compose` instead:
https://docs.docker.com/compose/releases/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2

But it's probably worth doing this in a separate PR.

**Review by jakubgs — APPROVED** [review]

**Comment by igor-sirotin** [comment]
I checked that it fixes https://github.com/status-im/status-go/issues/5980

With failing tests
image

The failure is properly reported:
image

**Review by igor-sirotin — APPROVED** [review]

**Comment by igor-sirotin** [comment]
There's another issue that we referenc `INTEGRATION_TESTS_DOCKER_UID` instead of `FUNCTIONAL_TESTS_DOCKER_UID` here:
https://github.com/status-im/status-go/blob/bf729f98a6c544f94f76792dfa5e2ce663b145ee/tests-functional/docker-compose.test.status-go.yml#L31

Which leads to running `status-backend` container with root rights.
image

This has to be fixed, but can be done separately.

**Pull request merged**